stats/stats-proto/proto/stats.proto

```diff
@@ -5,37 +5,26 @@ package blockscout.stats.v1;
 option go_package = "github.com/blockscout/blockscout-rs/stats";
 
 service StatsService {
-  rpc GetCounters(CountersRequest) returns (Counters);
-  rpc GetNewBlocks(ChartRequest) returns (ChartInt);
+  rpc GetCounters(GetCountersRequest) returns (Counters);
+  rpc GetLineChart(GetLineChartRequest) returns (LineData);
 }
 
-message CountersRequest {}
+message GetCountersRequest {}
 
-message PointInt {
+// All integers are encoded as strings to prevent data loss
+message Point {
   string date = 1;
-  uint64 value = 2;
+  string value = 2;
 }
 
-message ChartInt { repeated PointInt chart = 1; }
+message LineData { repeated Point chart = 1; }
```

Comment:
Nit: still why not `LineChart` if we are trying to `GetLineChart`? Wouldn't it be more consistent?